### ISSUE TITLE
Unnecessary padding warning

### ISSUE THREAD

**Issue by arasmussen** [chromakode/react-html-email#12]
`"Style property `padding` supplied to `Item`, in outlook: padding for p, div and a tags is not supported"`

The `Item` component adds the padding to a `<td />` so this warning seems unnecessary. Can we only give it when the padding is added to a p, div, or a tag?

**Comment by chromakode** [comment]
Good idea! This warning is not very smart; it comes from the scraped data and doesn't know about the component it is being called on. We should add a special case to the style validator that detects this warning and checks whether the component matches.
